account_financial_risk/models/res_company.py

```diff
@@ -19,13 +19,3 @@ class ResCompany(models.Model):
         "Useful when the flow comes from sales orders and the over-risk "
         "has already been allowed when confirming these.",
     )
-
-    # TODO: Remove when odoo compare values before to write
-    def write(self, vals):
```

Comment:
I would say this is still happening in v13, isn't it, @carlosdauden ?

Reply:
Now there are'nt stored fields, so there should be no heavy re-computations

Reply:
After refactorization

Reply:
After refactorization. I'll remove v12 too.